pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -276,7 +276,7 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 		return insufficientResources
 	}
 
-	if podRequest.MilliCPU > (nodeInfo.Allocatable.MilliCPU - nodeInfo.Requested.MilliCPU) {
+	if podRequest.MilliCPU > 0 && podRequest.MilliCPU > (nodeInfo.Allocatable.MilliCPU-nodeInfo.Requested.MilliCPU) {
```

Comment:
you are assuming request is higher than allocatable, when will that happen?

Reply:
In our case, we have a service to update node allocatable resources(up and down) to overcommit pods. So there's a chance requested could be higher than allocatable